src/main/resources/bidder-config/loopme.yaml

```diff
@@ -0,0 +1,27 @@
+adapters:
+  loopme:
+    enabled: false
+    endpoint: http://prebid-eu.loopmertb.com
+    pbs-enforces-gdpr: true
+    pbs-enforces-ccpa: true
+    modifying-vast-xml-allowed: true
+    deprecated-names:
+    aliases: {}
+    meta-info:
+      maintainer-email: support@loopme.com
+      app-media-types:
+        - banner
+        - video
+        - native
+      site-media-types:
+        - banner
+        - video
+        - native
+      supported-vendors:
+      vendor-id: 0
```

Comment:
When I take a quick look on vendor-list I found this record, so should we add `109` as vendor-id?
```json
    "109": {
      "id": 109,
      "name": "LoopMe Limited",
```